docs/config.md

```diff
@@ -28,68 +28,77 @@ UAEPassPRODAuthzEndpoint = "https://id.uaepass.ae/idshub/authorize"
 UAEPassPRODUserInfoEndpoint = "https://id.uaepass.ae/idshub/userinfo"
 UAEPassPRODTokenEndpoint = "https://id.uaepass.ae/idshub/token"
 UAEPassPRODLogoutEndpoint = "https://id.uaepass.ae/idshub/logout"
+
 ```
+The [documentation](https://docs.uaepass.ae/guides/authentication/web-application/endpoints) for the UAEPass include information regarding above endpoints.
 
-The [documentation](https://docs.uaepass.ae/guides/authentication/web-application/endpoints) for the UAEPass then 
-include information regarding above endpoints.
-
-Once Identity Server restarts, the bellow configurations will be getting reflected into `application-authentication.xml` 
-file located in the `<IS-HOME>/repository/conf/identity` directory from the above toml file 
-
-```xml
-   <AuthenticatorConfig name="UAEPassAuthenticator" enabled="true">
-      <Parameter name="UAEPassSTGAuthzEndpoint">https://stg-id.uaepass.ae/idshub/authorize</Parameter>
-      <Parameter name="UAEPassSTGUserInfoEndpoint">https://stg-id.uaepass.ae/idshub/userinfo</Parameter>
-      <Parameter name="UAEPassSTGTokenEndpoint">https://stg-id.uaepass.ae/idshub/token</Parameter>
-      <Parameter name="UAEPassSTGLogoutEndpoint">https://stg-id.uaepass.ae/idshub/logout</Parameter>
-      <Parameter name="UAEPassPRODAuthzEndpoint">https://id.uaepass.ae/idshub/authorize</Parameter>
-      <Parameter name="UAEPassPRODUserInfoEndpoint">https://id.uaepass.ae/idshub/userinfo</Parameter>
-      <Parameter name="UAEPassPRODTokenEndpoint">https://id.uaepass.ae/idshub/token</Parameter>
-      <Parameter name="UAEPassPRODLogoutEndpoint">https://id.uaepass.ae/idshub/logout</Parameter>
-   </AuthenticatorConfig>
```

Comment:
Don't we need these configs now?

Reply:
Here, it is mentioned how the configs we add to deployment toml file is shown in .xml file. IMO this info is redundant